jaseci_ai_kit/jac_nlp/jac_nlp/action_configs/bart_sum_action_config.py

```diff
@@ -49,20 +50,28 @@
                                     "name": "bart-sum-up",
                                     "defaultMode": 420,
                                 },
-                            }
+                            },
+                            {
+                                "name": "jac-nlp-volume",
+                                "persistentVolumeClaim": {"claimName": "jac-nlp-pvc"},
+                            },
                         ],
                         "containers": [
                             {
                                 "name": "bart-sum",
-                                "image": "jaseci/jac-nlp:latest",
-                                "command": ["bash", "-c", "source script/prod_up"],
+                                "image": "jaseci/jaseci-experiment:1.4.0.12",
```

Comment:
We don't need to use the jaseci-experiment docker image anymore. Let's change this to be `jac-nlp:1.4.0.18`

Reply:
Will need to change this for all the action_config.py BTW.